From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mack Straight <mack@discordapp.com>
Date: Tue, 1 Feb 2022 01:06:39 +0000
Subject: specify that we're not building electron

this way the android build works

diff --git a/chromium/scripts/build_ffmpeg.py b/chromium/scripts/build_ffmpeg.py
index e7d4e42fc3f26d633730e5635e94da42ad860d87..3a427363f9683b56a3ea5ea37af2f6d9e546261f 100755
--- a/chromium/scripts/build_ffmpeg.py
+++ b/chromium/scripts/build_ffmpeg.py
@@ -193,6 +193,7 @@ class AndroidApiLevels:
     # target OS.
     with open(os.path.join(tmp_dir, 'args.gn'), 'w') as args_gn_file:
       args_gn_file.write('target_os = "android"\n')
+      args_gn_file.write('is_electron_build = false\n')
       print('Created ' + os.path.realpath(args_gn_file.name))
 
     # Ask GN to generate build files.
